types/mparticle__web-sdk/index.d.ts

```diff
@@ -3,6 +3,45 @@ import { Batch } from "@mparticle/event-models";
 // Placeholder for Dictionary-like Types
 export type Dictionary<V = any> = Record<string, V>;
 
+// Rokt Manager Types
+export interface RoktPartnerAttributes {
+    [key: string]: string | number | boolean | undefined | null;
+}
+
+export interface RoktPartnerExtensionData<T> {
+    [extensionName: string]: T;
+}
+
+export interface RoktSelectPlacementsOptions {
+    attributes: RoktPartnerAttributes;
+    identifier?: string;
+}
+
+export type RoktPlacement = Record<string, unknown>;
+
+export interface RoktSelection {
+    close: () => void;
+    getPlacements: () => Promise<RoktPlacement[]>;
+}
+
+export interface RoktLauncher {
+    selectPlacements: (options: RoktSelectPlacementsOptions) => Promise<RoktSelection>;
+    hashAttributes: (attributes: RoktPartnerAttributes) => Promise<Record<string, string>>;
+}
```

Comment:
I realize this is just our launcher and not that actual Rokt launcher, which is here - https://github.com/ROKT/sdk-web/blob/main/packages/web-sdk/integrations/src/controller/launcher/integration-launcher.ts#L10

We should make tickets to support everything here

Reply:
I think I may have been overzealous in adding this. We technically don't expose this on the Rokt Manager side.